### DISCUSSION TITLE
Interval Type

### DISCUSSION THREAD

**Discussion by jrock2004** [polybar/polybar#2755] [Q&A] [answered]
I am trying to look in the documentation and it escapes me what type interval is. Is it seconds, milliseconds?

**Comment by patrick96** [comment, marked as answer]
For which module is this not specified? Generally it should be seconds.

**Reply by jrock2004** [reply]
I was thinking that all the modules would be the same. If not, I think we should at least document somewhere so people can look it up. If its there, could you point it to me as I would love to read it

**Reply by patrick96** [reply]
Every module has its own page on the [wiki](https://github.com/polybar/polybar/wiki)

And not all modules even have an interval option
